### DISCUSSION TITLE
[Question] [Docker] What is the correct way to perform "php artisen migrate" in Dockerized deployment

### DISCUSSION THREAD

**Discussion by parthghughri** [snipe/snipe-it#17558] [Q&A]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

As per the documentation [here](https://snipe-it.readme.io/docs/docker#updating-snipe-it-or-mysql), while upgrading snipeit docker container we need to perform:

1. Stop old container (snipeit)
2. Pull new container (snipe/snipe-it)
3. Rename old container (snipeit -> snipeit_old)
4. Start newly pulled container (with name snipeit)
5. execute this command 👇
> docker run snipeit php artisan migrate

The issue here is it says "snipeit:latest does not found locally". 
If I give name snipe/snipe-it (docker run snipe/snipe-it PHP artisen migrate) it is trying to run a new container from the snipe/snipe-it image.

If the container is already running then shouldn't it be with **exec** with additional **--force**
> docker exec snipeit PHP artisen migrate --force

Here --force to prevent it from giving this error
![image](https://github.com/snipe/snipe-it/assets/16538468/ebd5f437-92f8-4ba0-bb0a-c80e433a8e39)

Thanks in advance.

### Reproduction steps

1. Stop old container (snipeit)
2. Pull new container (snipe/snipe-it)
3. Rename old container (snipeit -> snipeit_old)
4. Start newly pulled container (with name snipeit)
5. execute this command 👇
> docker run snipeit php artisan migrate

### Expected behavior

Execute php artisen migrate command

### Screenshots

By running the command defined in the documentation:
![image](https://github.com/snipe/snipe-it/assets/16538468/ebd5f437-92f8-4ba0-bb0a-c80e433a8e39)

### Snipe-IT Version

v6.2.4-pre build 12343 (gb23ce6cfc)

### Operating System

Ubuntu

### Web Server

Apache

### PHP Version

8.1.2-1ubuntu2.14

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
